pkg/proto/protodep.toml

```diff
@@ -0,0 +1,23 @@
+proto_outdir = "./protodep"
```

Comment:
Is this file is generated by `protoc`? This looks like a Gopkg file, I suppose we can't use a `go.mod` file instead?

Reply:
Nope, the file is manually created and maintained and is specific to `protodep`.